onnxruntime/python/tools/transformers/models/llama/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Let's rename this script to `run_70b_model.sh` so that it is clear this script is just for the 70B model.